Adds a web service to check LRS if module is completed

bbeversdorf opened the pull request:
This allows the module to reach out to the LRS and checks if the specified module is completed for the user.

The xAPI Launch Link plugin does not immediately go and check for a status update from the LRS. Although the cron can be set to frequently check completion, this gives the module control when to check.

Commit:
Adds a web service to check LRS if module is completed

Reaches out to the LRS and checks if the specified module is completed for the specified user.

garemoko commented:
Thanks for this!

This is currently failing automatic tests. Can you look at those first before I review this?

Commit:
Minor Fixes for Travis CI

externallib.php
-Removed Extra Whitespace

db/services.php
-Corrected “Expected MOODLE_INTERNAL check or config.php inclusion”

Commit:
Minor Fixes for Travis CI

externallib.php
-Removed Extra Whitespace

db/services.php
-Corrected “Expected MOODLE_INTERNAL check or config.php inclusion”

Commit:
Merge pull request #1 from bbeversdorf/develop

Minor Fixes for Travis CI

Commit:
Update services.php

bbeversdorf commented:
I didn't have my forked branches setup the way I wanted (and inadvertently committed to master), so I recreated my master branch. I will open another PR.
